javaagent/Dockerfile

```
@@ -1,4 +1,4 @@
-FROM eclipse-temurin:11-jre
+FROM eclipse-temurin:21-jre
```

you should be able to set this to 17 now
```suggestion
FROM eclipse-temurin:17-jre
```